homework1/src/main/java/com/epam/spring/homework1/config/BeanConfig.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.spring.homework1.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@ComponentScan("com.epam.spring.homework1.beans")
```

Comment:
It is much safer way to use class as a reference for a package. Please explore `@ComponentScan` annotation